ansible/roles/postgresql/tasks/databases.yml

```diff
@@ -29,6 +29,7 @@
     encrypted: yes
     name: "{{ item.0.user }}"
     password: "{{ item.0.password }}"
+    role_attr_flags: "CREATEDB,NOSUPERUSER"
```

Comment:
The `postgresql` role is used in production. Isn't this giving excessive privileges to the omero db user?